wiki/Replay/en.md

```diff
@@ -0,0 +1,303 @@
+
+A [Cinema][Cinema wikilink] replay will:-
+
+- hide all the replay options,
+- disable gameplay access,
+- hide all the skin elements of the game mode,
+- enable the background image/video,
+- play only the pass storyboard,
+- set to 0% background dim, and
+- play the hit sounds on timing points where the notes would be completed.
+
+To skip the starting/ending point of the play, press the `Spacebar` key.
+
+### Discussion
+
+The button **only appears when an in-game osu! account was connected to Bancho and the difficulty/beatmap can be found in the osu! beatmaplist (ranked or otherwise)**.
+Shows flowing right-to-left comments left by the community on the beatmap.
+
+Click on the `Discussion` button to show the options available.
+
+![Replay discussion screen][Replay discussion]
+
+_Replay with discussion options in view_
+
+Click on `Show Comments` to enable flowing comments regarding the beatmap(set) at the middle of the screen.
+An active osu!supporter account will have an extra `Colour` button which allows new comment to be coloured.
+
+![Adding a comment by a player/mapper][Replay discussion comment]
+
+_Making a comment_
+
+Click on `Click to add a comment at the current time!` to comment on the current difficulty/beatmap.
+The replay will be paused, darken, and with an osu!logo at the foreground center until the comment was completed (usually, `Enter` key) or cancelled (usually, `Esc` key).
+
+![Replay with discussion enabled][Replay discussion example]
+
+_Commentary by the community at the middle, mapper at the top in blue_
+
+To filter the comments by difficulty, enable `Difficulty (#)`.
+To filter the comments by the overall beatmapset, enable `Song (#)`.
+When watching a top play from the _Global leaderboard_, enable `Player (#)` to show comments from the player itself.
+
+Mapper's comment will be placed above the flowing comments with a light-blue text and an jump-out/jump-in animation instead.
+Depending on resolution, it may be obstructed by a **UNRANKED** notice when watching an Auto replay.
+To remove the **UNRANKED** notice, either watch a top replay from the _Global leaderboard_, or a local replay from the _Local leaderboard_ without the Auto mod used.
+
+When watching a top replay, the player of the replay's comment(s) will be shown _below_ the screen instead, and will only be shown in this replay only.
+
+By default without active osu!supporter, comments will be coloured based on chat colour (normal accounts will be white) except for mapper's comment(s) (light-blue).
+
+### Collections
+
+Add this beatmap(set) to _Collections_.
+This will open up the Collections screen to put the current beatmap(set) into specific sections or rename the sections.
+
+The replay will not pause and continue at the background while in this screen.
+
+### Playback Speed
+
+Click to adjust the `1x`/`2x`/`0.5x` playback speed of the replay to `2x`/`0.5x`/`1x` playback speed.
```

Comment:
I get what this is trying to stay but I think it could be phrased better... reword this?